addons/web_editor/tools.py

```diff
@@ -134,28 +133,39 @@ def get_video_embed_code(video_url):
     return Markup('<iframe class="embed-responsive-item" src="%s" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowFullScreen="true" frameborder="0"></iframe>') % data['embed_url']
 
 
-def get_video_thumbnail(video_url):
-    """ Computes the valid thumbnail image from given URL
+def get_video_thumbnail_url(video_url):
+    """ Computes the valid thumbnail image URL from given URL
         (or None in case of invalid URL).
     """
     source = get_video_source_data(video_url)
     if source is None:
         return None
 
-    response = None
+    thumbnail_url = None
     platform, video_id = source[:2]
-    with contextlib.suppress(requests.exceptions.RequestException):
```

Comment:
Shouldn't this behavior be kept in the new implementation of `get_video_thumbnail` ?
Not doing it changes the old behavior.